### PR TITLE
added angular icon in the source pane for angular projects

### PR DESCRIPTION
**Pull request by borian**
Associated Issue: #4733

### Summary of Changes

added angular icon in the source pane for angular projects

light
dark

### CODE REVIEW

File: src/components/shared/Svg.css [hunk 1]
```diff
@@ -115,3 +115,14 @@ html .arrow.expanded svg {
 .theme-dark .webpack {
   opacity: 0.5;
 }
+
+.managed-tree .tree-node .angular svg {
```
Comment (James300) [inline src/components/shared/Svg.css]: Any reason this isn't just `.angular`, I understand the difference but I'm curious if this is safer / more maintainable / something else?

Comment (borian) [reply src/components/shared/Svg.css]: Sorry, I only mentioned this in the issue https://github.com/devtools-html/debugger.html/issues/4733.
The icon is already used in the call stack view, thats why I added specific styling for the source pane

**Comment by jasonLaster** [comment]
@borian can we replace `ng://` and `Angular`

**Comment by borian** [comment]
@jasonLaster shure, I changed the name and squashed the commit
ng-light
ng-dark

**Comment by jasonLaster** [comment]
Thanks i made a couple of tweaks including the shorter classname, hopefully we'll be okay :)

**Comment by borian** [comment]
This css will add additional styling to the icon used in the call stack view.
But it will only be 3px larger, i think 😄

**Pull request merged**